lib/include/dlisio/lis/protocol.hpp

```diff
@@ -11,6 +11,16 @@
 
 namespace dlisio { namespace lis79 {
 
+/** Size of LIS type
```

Comment:
1. Why this function is in protocol.hpp, not in any typey-stuff?
2. Maybe more explicit clarification in name or in docs is needed to note that it's src (specification) type, not dst (your computer) type. One can "store LIS type" on both, so it's too vague.